doc/programs.md

```diff
@@ -102,6 +102,7 @@
 - MODEMS, modems gragon.
 - MSPLIT, split a file into smaller parts.
 - MTBOOT, make bootable tapes.
+- MUDDLE, MDL interpreter.
```

Comment:
Since the executable is named TS MDL, I suppose this could be "MDL, Muddle interpreter."  Or whatever the proper name is.